src/utils/calculateStakingRewards.ts

```diff
@@ -1,4 +1,4 @@
-const calculateStakingRewards = (totalAtStake) => {
+const calculateStakingRewards = (totalAtStake: number) => {
```

Comment:
Shouldn't we explicitly declare a return type here?

Reply:
Yes, fixed. Thx :)